### ISSUE TITLE
feature: Python 

### ISSUE THREAD

**Issue by Hydra-hcr** [Smug246/Luna-Grabber#908]
### What Feature?

which version i choose please 
![discord luna grabber](https://github.com/user-attachments/assets/e84dfdbc-7136-4f80-9266-f7a79509eae1)

### Add Screenshots (If Applicable)

![image](https://github.com/user-attachments/assets/89001731-e441-44e2-bf23-9042e46785c5)

### Check the following before submitting (If Applicable)

- [X] I have checked the existing [issues](https://github.com/Smug246/Luna-Grabber/issues)
- [ ] I want to work on this feature

**Comment by NFS11** [comment]
press 1?

**Comment by NFS11** [comment]
you need 3.11 if u wanna use nuitka compiler. but you can still get it to work anyway

**Comment by sdas1247** [comment]
dont use this , pls delete the file it steals ur crypto walle i ran the file setup.bat and it stole my freaking 365$

**Comment by 42400** [comment]
> ne l'utilisez pas, veuillez supprimer le fichier, il vole votre portefeuille cryptographique, j'ai exécuté le fichier setup.bat et il m'a volé mes 365 $

ces un truc maveillant alors se LUNA GRABEUR ?

**Comment by sdas1247** [comment]
yes it is malicious it stole my 365$ , it steals your wallet rather than stealing someone else wallet dont download it please

**Comment by 42400** [comment]
Merciii beaucoup de m’avoir prévenu !

**Comment by NFS11** [comment]
@sdas1247 that's crazy... it should never steal stuff like that. I have been using it for quite a while and nothing has happened to me. you may have used a fake of some sort. someone else has been hacked downloading a fake. this isn't the first time something like this has happened

**Comment by 42400** [comment]
Une sorte de faux c’est à dire ?

**Comment by InfernoTsugikuni** [comment]
> yes it is malicious it stole my 365$ , it steals your wallet rather than stealing someone else wallet dont download it please

i can confirm it doesnt happen, looked thru the code and everything is fine, you probably dont really know how malware works but it doesnt steal your wallet at all

**Comment by NFS11** [comment]
@sdas1247 so basically youre saying that as soon as you opened setup.bat it immediatly stole $365 from your crypto? that shouldnt really happen ive known this tool and ran it multiple times on old pcs i own and in vms. you may have downloaded something else. either way have you tried getting your money back? plus not not to mention 365 is a really specific number

**Comment by NFS11** [comment]
@42400  je suppose que oui. J'ai utilisé l'outil plusieurs fois sur d'anciens ordinateurs de test que je possède et sur des machines virtuelles et rien de malveillant ne m'est arrivé, comme le vol de mes données ou de mon argent. il y avait en fait quelqu'un d'autre qui avait une situation similaire et qui a déclaré que ses données avaient été générées et exécutables d'une manière ou d'une autre. mais je peux confirmer qu'il ne vole aucune forme d'argent ou de crypto à moins que vous n'ouvriez le .exe généré et qu'il envoie ces données à votre propre webhook sur votre propre serveur. les traductions de ce commentaire peuvent être inutiles puisque j'utilise Google Translate.
